packages/connect-node/src/grpc-http-transport.ts

```diff
@@ -0,0 +1,408 @@
+              method: "POST",
+              path: endpoint.pathname,
+              signal: req.signal,
+              ...options.httpOptions,
+            });
+
+            const responsePromise = new Promise<http.IncomingMessage>(
+              (resolve, reject) => {
+                stream.on("response", (res) => {
+                  resolve(res);
+                });
+                stream.on("error", reject);
+              }
+            );
+            const responseTrailer = defer<Headers>();
+            const responseHeader = responsePromise.then((res) =>
+              nodeHeaderToWebHeader(res.headers)
+            );
+            const conn: StreamingConn<I, O> = {
+              ...req,
+              responseHeader,
+              responseTrailer,
+              closed: false,
+              async send(message: PartialMessage<I>) {
+                if (stream.writableEnded) {
+                  throw new ConnectError(
+                    "cannot send, stream is already closed"
+                  );
+                }
+                const enveloped = encodeEnvelope(
+                  messageFlag,
+                  serialize(normalize(message))
+                );
+                await write(stream, enveloped);
+              },
+              async close() {
+                if (stream.writableEnded) {
+                  throw new ConnectError(
+                    "cannot close, stream is already closed"
+                  );
+                }
+                this.closed = true;
+                await end(stream);
+              },
+              async read(): Promise<ReadableStreamReadResultLike<O>> {
+                const response = await responsePromise;
+                assert(
+                  typeof response.statusCode == "number",
+                  "http1 client response is missing status code"
+                );
+                validateResponse(
+                  useBinaryFormat,
+                  response.statusCode,
+                  await responseHeader
+                );
+                try {
+                  const result = await readEnvelope(response);
+                  if (result.done) {
+                    const trailer = mapResponseTrailers(response.trailers);
+                    validateGrpcStatus(trailer);
+                    responseTrailer.resolve(trailer);
+                    return {
+                      done: true,
+                      value: undefined,
+                    };
+                  }
+                  return {
+                    done: false,
+                    value: parse(result.value.data),
+                  };
+                } catch (e) {
+                  throw connectErrorFromNodeReason(e);
+                }
+              },
+            };
+            return Promise.resolve(conn);
+          } catch (e) {
+            throw connectErrorFromNodeReason(e);
+          }
+        }
+      );
```

Comment:
runStreaming() takes three arguments. You're missing the third one here, you need to pass `options.interceptors`.

Let's make the argument optional: https://github.com/bufbuild/connect-web/pull/369